### PR TITLE
qemu_vm: add backing_file option for block_stream.

### PR DESCRIPTION
**Pull request by vivianQizhu**
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

File: virttest/qemu_monitor.py [hunk 1]
```diff
@@ -2033,14 +2033,15 @@ def live_snapshot(self, device, snapshot_file, **kwargs):
             kwargs.update({"mode": "absolute-paths"})
         return self.cmd("blockdev-snapshot-sync", kwargs)
 
-    def block_stream(self, device, speed=None, base=None,
+    def block_stream(self, device, speed=None, base=None, backing_file=None,
```
Comment (apahim) [inline virttest/qemu_monitor.py]: Please add new parameters at the end of the list.

File: virttest/qemu_vm.py [hunk 2]
```diff
@@ -4331,7 +4331,7 @@ def live_snapshot(self, base_file, snapshot_file,
 
         return current_file
 
-    def block_stream(self, device, speed, base=None, correct=True):
+    def block_stream(self, device, speed, base=None, backing_file=None, correct=True):
```
Comment (apahim) [inline virttest/qemu_vm.py]: Ditto.

**Comment by vivianQizhu** [comment]
@apahim Thanks, updated.

**Comment by vivianQizhu** [comment]
2017-06-13 14:43:13,458 test L0750 INFO | PASS 1-smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.my-provider.block_stream.backingfile_option.absolute_path
2017-06-13 14:46:01,682 test L0750 INFO | PASS 2-smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.my-provider.block_stream.backingfile_option.relative_path

14:40:41 DEBUG| Send command: {'execute': 'block-stream', 'arguments': {'device': u'drive_image1', 'base': u'/home/kvm_autotest_root/images/rhel74-64-virtio.qcow2', 'speed': 0, 'backing-file': u'/home/kvm_autotest_root/images/rhel74-64-virtio.qcow2'}, 'id': 'radW84fZ'}

**Comment by pezhang** [comment]
smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.spice.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.block_stream.backingfile_option.absolute_path: PASS (215.84 s)

smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.spice.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.block_stream.backingfile_option.relative_path: PASS (215.09 s)

Looks good to me.
Acked-by: pezhang@redhat.com

**Pull request merged**